js/ui/file_manager/ui.file_manager.adaptivity.js

```diff
@@ -10,6 +10,8 @@ import SplitterControl from "../splitter";
 const window = getWindow();
 const ADAPTIVE_STATE_SCREEN_WIDTH = 573;
 
+const DRAWER_PANEL_CONTENT_INITIAL = "dx-drawer-panel-content-initial";
```

Comment:
I think that the class name `"dx-drawer-panel-content-initial"` should have the prefix `"dx-filemanager"`, because this class name is used for the FileManager widget, and not for a standalone Drawer widget.